ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal.tsx

```diff
@@ -193,29 +167,125 @@ export function AssetPickerModal({
   }, []);
 
   // filter and exclude ERC1155
-  const collectionDataFiltered = (collectionsData as Collection[])
-    .map((collection) => {
-      return {
-        ...collection,
-        nfts: collection.nfts.filter(
-          (nft) => nft.standard !== TokenStandard.ERC1155,
-        ),
-      };
-    })
-    .filter((collection) => collection.nfts.length > 0);
+  const collectionDataFiltered = (collectionsData as Collection[]).filter(
+    (collection) => collection.nfts.length > 0,
+  );
 
   const handleSearch = (query: string) => {
     setSearchQuery(query);
   };
 
   const handleAssetChange = (token: Token) => () => {
     onAssetChange(token);
-    setSelectedToken(token.address);
     onClose();
   };
 
   const defaultActiveTabKey = asset?.type === AssetType.NFT ? 'nfts' : 'tokens';
 
+  const AssetList = useCallback(() => {
```

Comment:
Can this be split out into a separate component?

Reply:
Great callout – done.